File: python/regularize.py
import pandas as pd

# Import dataset
df = pdf.read_csv('data.csv')
X = df.iloc[:, :-1].values
y = df.iloc[:, -1].values

# Split into training and test sets
from sklearn.cross_validation import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)

# Fit regularized linear regression to training set
from sklearn.linear_model import ElasticNet
model = ElasticNet(alpha=0.1, l1_ratio=0.5)
model.fit(X_train, y_train)

# Predict test set
y_pred = model.predict(X_test)

File: r/regularize.r
# Import dataset
df = read.csv('data.csv')

# Create Boolean vector with random split
library(caTools)
split = sample.split(df$y, SplitRatio=0.8)

# Split into training and test set
train = subset(df, split == TRUE)
test = subset(df, split == FALSE)

# Generalized Linear Models Lasso and Elastic Net (glmnet)
# contains efficient procedures for fitting lasso or 
# elastic-net regularization for linear regression,
# multinomial regression models, Poisson regression,
# multiple-response Gaussian, and the Cox model.
install.packages('glmnet')
library(glmnet)

# Fit regularized linear regression to training set
y_col = ncol(train)
model = glmnet(x=train[,-y_col], y=train[,y_col], alpha=0.5)

# Predict test set
y_pred = predict(model, newdata=test)